app/views/groups/manage/existing_team_members.scala.html

```diff
@@ -17,42 +17,72 @@
 @import config.AppConfig
 @import controllers.routes
 @import views.html.main_layout
-@import utils.ViewUtils._
+@import uk.gov.hmrc.agentmtdidentifiers.model.AccessGroup
 
 @this(
-    layout: main_layout,
-    p: p, h1: h1, a: a,
-    caption: caption,
+    layout: main_layout,table: table,
+    p: p, h1: h1, a: a, caption: caption,
     link_as_button: link_as_button,
-    table: table,
     submit_button: submit_button,
-    textbox: textbox,
-    checkBox: checkBox
+    textbox: textbox, checkBox: checkBox,
+    submit_button_group: submit_button_group
 )
 
-@(teamMembers: Seq[TeamMember], groupName: String, continueUrl: String)(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
+@(
+    teamMembers: Seq[TeamMember],
+    form: Form[SearchFilter],
+    group: AccessGroup,
+)(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
 
     @tableRows = @{
         teamMembers.map(teamMember =>
             Seq(
                 Html(teamMember.name),
                 Html(teamMember.email),
-                Html(displayTeamMemberRole(teamMember.credentialRole.getOrElse("")))
+                Html(teamMember.credentialRole)
```

Comment:
You undid the change to the team member role display with one of your merges to main

Reply:
ah. test failing!